ui/sitemaps.md

```diff
@@ -584,30 +584,40 @@ If the parameter is not provided, the default is to display the Item.
 Visibility syntax:
 
 ```java
-visibility=[item_name operator value, item_name operator value, ... ]
+visibility=[item_name operator value, item_name operator value AND item_name operator value, ... ]
 ```
 
+You set as many conditions as you want.
+
+A condition can be a single comparison or a combination of several comparisons separated by the AND operator.
+You can use as many AND-separated comparisons as you want.
```

Comment:
Maybe combine this sentence with line 590?

Reply:
I removed this sentence, I believe it is already clear with the previous sentence.